dev_guide/templates.adoc

```diff
@@ -105,13 +86,77 @@ $ osc process -f examples/sample-app/application-template-dockerbuild.json -v AD
 
 ====
 
-The JSON string can either be redirected to a file or applied directly by piping the process output to the `apply` command, as shown in the following example.
+The JSON string can either be redirected to a file or applied directly by piping the process output to the `create` command:
 
 .Redirecting JSON String
 ====
 
 ----
-$ osc process -f examples/sample-app/application-template-dockerbuild.json -v ADMIN_USERNAME=root,MYSQL_DATABASE=admin | osc apply -f -
+$ osc process -f examples/sample-app/application-template-dockerbuild.json -v ADMIN_USERNAME=root,MYSQL_DATABASE=admin | osc create -f -
+----
+
+====
+
+=== Creating from Template
+
+To create an configuration from a template using the Management Console:
+
+1. Click on the *Create+* button:
++
+====
+
+image:console_create.png["Console Create"]
+====
+
+2. Select a template from the list of templates in your project, or provided by OpenShift:
```

Comment:
Maybe:

s/OpenShift/the global template library/

Side note: Ultimately we should have an `admin_guide` topic about cluster admins adding templates to the global library in the "openshift" project; then "global template library" here could link to that.